### PR TITLE
log_interval 5

### PR DESCRIPTION
**Pull request by jaimepedretp**
log_interval 10 to 5

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
         'num_episodes': 1250,  # Number of training episodes
         'lr': 1e-2,  # Learning rate
         'gamma': 0.99,  # Discount rate
-        'log_interval': 10,  # controls how often we log progress
```
Comment (ziritrion) [inline main.py:14]: I don't like odd numbers, but I'll accept it anyway

**Review by eldarsilver — APPROVED** [review]
Seal of approval!!

**Review by xeviknal — CHANGES_REQUESTED** [review]
I don't like this. This is tooo small.

**Review by ziritrion — COMMENTED** [review]
I can't test it, my dog ate my laptop

**Review by xeviknal — APPROVED** [review]
It looks good. I do prefer to see updates quite more often.

**Review by ziritrion — APPROVED** [review]
I punched my dog and he puked my laptop. I tested it and I am now a millionare. I approve.

**Comment by xeviknal** [comment]
selfie-0

**Comment by ziritrion** [comment]
I push the green button ^_^

**Pull request merged**